src/components/modules/dropdown/index.tsx

```diff
@@ -0,0 +1,22 @@
+'use client';
+
+import { DropdownOption, DropdownOptionsWrapper, DropdownWrapper } from './dropdown.styles';
+import type { DropdownProps } from './dropdown.types';
+
+import { Button } from '@/components/elements/button';
+
+export const Dropdown = ({ children, items, ...buttonProps }: Component<DropdownProps>) => {
```

Comment:
1. Let's make sure to add some accessibility also here, please.
2. The dropdown component uses the caret icon by default but is not here.
3. There is an issue with the item's alignment when the text is large
4. It seems that some shadow or background doesn't follow the border radius of the button
![Screenshot 2024-02-22 at 10 21 36](https://github.com/bejamas/httptoolkit-website/assets/22168189/0234bcd4-d395-40e3-9a8e-171a5d292f31)

5. The component doesn't follow the design styles
![Screenshot 2024-02-22 at 10 27 00](https://github.com/bejamas/httptoolkit-website/assets/22168189/e101ff2e-4b83-45f1-a0b0-8b51d03f822c)

Reply:
@RBoraurito there is still some inconsistency